### PR TITLE
Added a new branch with CPS-OOD specific instructions under first_steps.

### PR DESCRIPTION
**Pull request by marianalevineu**
This branch was created for specific CPS-OOD JuypterLab instructions.

### CODE REVIEW

File: docs/source/first_steps/cps_ood.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+Here you can create and edit files. 
+
+#. 3. Open a new Python Notebook session from the Launcher menu by clicking the **Python 3 (ipykernel)**. 
+
+.. image:: /images/cps-ood-jupyterlab-ipykernel-launcher.png
+ :width: 400
+ :alt: show inside ipykernel.
+
+A new file will be created inside your directory called **Untitled.ipynb**. You can rename it by right-click + Rename option.
```
Comment (manasvita) [inline docs/source/first_steps/cps_ood.rst]: You can rename it by right-clicking on it and using the Rename option

File: docs/source/first_steps/cps_ood.rst [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+This Python notebook has ready-to-use Python packages needed for your class.
+
+.. note:: 
+  **Permission Denied errors:**
+  Do not attempt to create, edit or write files that are outside of your personal student directory. Most "Permission Denied" errors are due to directories or files having read-only access permissions. 
+
+Step 3 - submit class assignments
+=================================
+Due to the write-only access permissions on the **assignments** directory, it is required to use the command line interface (Linux Terminal) to submit assignments. **Using other methods currently does not work**.
```
Comment (manasvita) [inline docs/source/first_steps/cps_ood.rst]: Currently, using other methods, such as OOD interface (?) does not work

Comment (rschroter) [reply docs/source/first_steps/cps_ood.rst]: @marianalevineu looks like this hasn't been corrected (do->does)

**Review by rschroter — COMMENTED** [review]
everything looks good!

**Comment by marianalevineu** [comment]
thanks @rschroter can you try to merge it? I'm unable to...

**Review by rschroter — APPROVED** [review]
looks good, will merge!

**Pull request merged**